packages/widgets/src/menu.ts

```diff
@@ -1009,6 +1012,13 @@ export namespace Menu {
      * menu to be added as the last child of the host.
      */
     ref?: HTMLElement;
+
+    /**
+     * The alignment of the menu.
+     *
+     * The default is `'left'` unless the document `dir` attribute is `'rtl'`
+     */
+    align?: 'left' | 'right';
```

Comment:
To make this more future-proof, should we call it `horizontalAlignment` instead? So that we can add `verticalAlignment` if we need it later?

Otherwise it looks great!

Reply:
Good point! I just updated that.